packages/access-api/src/models/delegations.js

```diff
@@ -5,19 +5,24 @@ import {
 } from '@web3-storage/access/encoding'
 
 /**
- * @typedef {import('@web3-storage/access/src/types').DelegationTable} DelegationRow
+ * @typedef {import('@miniflare/r2').R2Bucket} R2Bucket
+ */
+
+/**
+ * @template {import('../types/access-api-d1').DelegationsV2Table | import('../types/access-api-d1').DelegationsV3Table} DelegationRow
  * @typedef {Omit<DelegationRow, 'inserted_at'|'updated_at'|'expires_at'>} DelegationRowUpdate
  */
 
 /**
- * @typedef Tables
- * @property {DelegationRow} delegations_v2
+ * @typedef V2Tables
```

Comment:
Duplicate with `AccessApiD1TablesV2`.
Either way, I will remove 'v2' tables now that there is a working `V3Tables`